props/prop_base.h

```diff
@@ -542,17 +542,32 @@ class PropBase : CommandParser, Looper, protected SaberBase {
   bool blade_detected_ = false;
 #endif
 
+  // Prevent BLADE ID: from spamming console with every BLADE_SCAN_MILLIS
+  bool blade_id_should_report_ = true;
+
   // Measure and return the blade identifier resistor.
   float id() {
     EnableBooster();
     BLADE_ID_CLASS_INTERNAL blade_id;
     float ret = blade_id.id();
+#ifdef BLADE_ID_SCAN_MILLIS
+    if (blade_id_should_report_) {
```

Comment:
While this work, it is not the best way to change the behavior of a function.
We should do one of:

1: add an argument to id() that specifies whether to do it quietly or not
2: split id into two functions, one quiet one and one noisy one, then call the right function for the right job

Using a global variable to change how a function works should only be used when 1 and 2 doesn't work for some reason.

Reply:
ok, how about now? simpler approach.